app/Filament/Resources/UserResource/Pages/EditUser.php

```diff
@@ -76,7 +81,10 @@ public function form(Form $form): Form
                             ->string()
                             ->maxLength(255),
                         Toggle::make('is_external')
-                            ->label('User can only login via Single Sign-On (SSO)'),
+                            ->label('User can only login via Single Sign-On (SSO)')
+                            ->live()
+                            ->afterStateUpdated(fn (Toggle $component, $state) => $state ? null : (($azureSsoSettings || $googleSsoSettings) ? $component->state(true) && $this->mountAction('showSSOModal') : null))
+                            ->default(($azureSsoSettings || $googleSsoSettings)),
```

Comment:
There should be no need for the `default` call here since this will be populated by the actual value on the User Model

Reply:
Done removed default()